conjur/client.py

```diff
@@ -166,11 +166,12 @@ def whoami(self):
         """
         return self._api.whoami()
 
-    def list(self):
+    # Constraints remain an optional parameter for backwards compatibility in the SDK
+    def list(self, list_constraints=None):
```

Comment:
list is a python saved keyword doesn;t the IDE show warning about it?

Reply:
nope it doesn't it is also used in the Client and does show some sort of warning.